### PR TITLE
Homepage Wrong Node Section

### PR DESCRIPTION
**Pull request by ericyliu**
1. replaced typing component with new one that can take commands and supports multiple lines
2. added Berlin Type font (free)

wrong-node
wrong-node-mobile

### CODE REVIEW

File: website2/src/components/terminal/styles.module.css [hunk 1]
```diff
@@ -14,11 +14,20 @@
 .content {
   border: 1px solid var(--color-black);
   padding: 24px;
+  overflow: scroll
```
Comment (alexdavid) [inline website2/src/components/terminal/styles.module.css:17]: I think this should be auto, right?

```suggestion
  overflow: auto;
```

File: website2/src/components/typingText/index.tsx [hunk 2]
```diff
@@ -3,18 +3,38 @@
...
 interface Props {
   prependedText: string;
-  texts: string[];
+  commands: Command[];
 }
 
-export const TypingText = ({ prependedText, texts }: Props) => {
-  const [termState, setTermState] = useState({
-    deleting: false,
+export const TypingText = ({ prependedText, commands }: Props) => {
```
Comment (alexdavid) [inline website2/src/components/typingText/index.tsx:27]: This change is super cool!

**Review by alexdavid — APPROVED** [review]
This looks great!

File: website2/src/components/typingText/index.tsx [hunk 3]
```diff
@@ -3,18 +3,38 @@
...
     idx: 0,
-    text: "",
+    lines: [],
+    currentLine: "",
   });
-  const currentExample = texts[termState.idx];
+  const currentCommand = commands[termState.idx];
   useEffect(() => {
     let timeout: NodeJS.Timeout;
     const type = (delay: number, newState: Partial<typeof termState>) => {
```
[the remark this answers is not in the document]
Comment (ericyliu) [reply website2/src/components/typingText/index.tsx:40]: good point

**Pull request merged**